Sources/ProjectDescription/TuistConfig.swift

```diff
@@ -20,3 +22,108 @@ public class TuistConfig: Codable {
         dumpIfNeeded(self)
     }
 }
+
+public struct TemplateString: Encodable, Decodable, Equatable {
```

Comment:
Considering this is a public type, I'd move it into its own Swift file, `TemplateString.swift`.